third_party/prost/0001-Patch-to-generate-typesafe-Sender-and-Receiver.patch

```diff
@@ -0,0 +1,51 @@
+From df8d3e37563d373ac38cc90f75f97416ba766b61 Mon Sep 17 00:00:00 2001
+From: Daan de Graaf <daandegraaf9@gmail.com>
```

Comment:
Should we also change this to `daagra@google.com`?

Reply:
Changed.